virtual_ecosystem/models/animal/scaling_functions.py

```diff
@@ -36,35 +36,6 @@ def damuths_law(mass: float, terms: tuple) -> int:
     return ceil(terms[1] * mass ** terms[0])
 
 
-def metabolic_rate_energy(
```

Comment:
Same here. This function is removed but not added anywhere else and, also, not related with the purpose of the PR. Is it an error or am I missing anything?